frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/constants.ts

```diff
@@ -2,11 +2,11 @@ export const CD_SIZE = 'size';
 export const CD_STORAGE_CLASS = 'storageClass';
 
 export const WINTOOLS_CONTAINER_NAMES = {
-  openshift: 'virtio-win-container',
-  ocp: 'virtio-win-container',
-  online: 'virtio-win-container',
-  dedicated: 'virtio-win-container',
-  azure: 'virtio-win-container',
+  openshift: 'cnv-tech-preview/virtio-win',
```

Comment:
please pull https://access.redhat.com/containers/#/registry.access.redhat.com/container-native-virtualization/virtio-win not the older cnv-tech-preview one. 

(but please verify)

Reply:
@dankenigsberg I have update the PR with accordance to the current result of the mailing list discussion. Verified functionality.